examples_utils/benchmarks/run_benchmarks.py

```diff
@@ -196,7 +196,7 @@ def run_benchmark_variant(
 
     # Check if benchmark calls poprun and might require some env variables,
     # and modify if needs be
-    variant_command = check_poprun_env_variables(benchmark_name, variant_command)
+    check_poprun_env_variables(benchmark_name, variant_command)
```

Comment:
Is there a way this check could be run earlier? The scenario I'm imagining is trying to run 10 benchmarks, and the 5th fails after 10 hours. Could it validate all the commands first and then start running them?

Reply:
Good point, I think it should be possible. I'll make the change.

Reply:
It was possible indeed.